src/reference/config/writing-platform-configs.rst

```diff
@@ -13,6 +13,8 @@ Writing Platform Configurations
    - :cylc:conf:`global.cylc[platforms]`
    - :cylc:conf:`global.cylc[platforms][<platform name>]install target`
 
+.. include:: user-platform-warning.rst
+
```

Comment:
I would remove the include file and add something like the following to the end of the "What Are Platforms" section below:

```rst
If you are working on an institutional network then
platforms may already have been configured for you.

To see available platforms::

   cylc config -i [platforms]
```

(the remainder of the include file gives information contained in this section and point back at this section)